css/css-grid/alignment/grid-align-baseline-vertical.html

```diff
@@ -95,45 +94,28 @@
   <div data-expected-width="100" data-offset-x="0" class="firstRowFourthColumn" style="width: 100px;"></div>
 </div>
 
-<div id="results"></div>
+<div id="log"></div>
 
 <script>
-function additionalBaselineChecks()
-{
-    var baseline1 = document.getElementById("baseline1");
-    var baseline2 = document.getElementById("baseline2");
-    var baseline3 = document.getElementById("baseline3");
-    logIfFalse(baseline1.offsetLeft == baseline2.offsetLeft,
-               "baseline1 and baseline2 should have the same baseline.");
-    logIfFalse(baseline3.offsetLeft + baseline3.offsetWidth > 75,
-               "baseline3 should be above the center align-itemsed item.");
-
-    var baseline4 = document.getElementById("baseline4");
-    var baseline5 = document.getElementById("baseline5");
-    logIfFalse(baseline4.offsetLeft + baseline4.offsetWidth == baseline5.offsetLeft + baseline5.offsetWidth,
-               "baseline4 and baseline5 should be right aligned with each other.");
-    logIfFalse(baseline4.offsetLeft + baseline4.offsetWidth < 100,
-               "baseline4 and baseline5 should be below baseline6.");
-
-    var baseline6 = document.getElementById("baseline6");
-    var baseline7 = document.getElementById("baseline7");
-    logIfFalse(baseline6.offsetLeft == baseline7.offsetLeft,
-               "baseline6 and baseline7 should have the same baseline.");
-
-    var baseline8 = document.getElementById("baseline8");
-    var baseline9 = document.getElementById("baseline9");
-    logIfFalse(baseline8.offsetLeft == baseline9.offsetLeft,
-               "baseline8 and baseline9 should be left aligned with each other.");
+  var baseline1 = document.getElementById("baseline1");
+  var baseline2 = document.getElementById("baseline2");
+  var baseline3 = document.getElementById("baseline3");
+  var baseline4 = document.getElementById("baseline4");
+  var baseline5 = document.getElementById("baseline5");
+  var baseline6 = document.getElementById("baseline6");
+  var baseline7 = document.getElementById("baseline7");
+  var baseline8 = document.getElementById("baseline8");
+  var baseline9 = document.getElementById("baseline9");
+
+
+  test(() => {assert_equals(baseline1.offsetLeft, baseline2.offsetLeft, "baseline1 and baseline2 should have the same baseline;")}, "Additional Check 1");
+  test(() => {assert_greater_than(baseline3.offsetLeft + baseline3.offsetWidth, 75, "baseline3 should be above the center align-itemsed item;")}, "Additional Check 2");
+  test(() => {assert_equals(baseline4.offsetLeft + baseline4.offsetWidth, baseline5.offsetLeft + baseline5.offsetWidth, "baseline4 and baseline5 should be right aligned with each other;")},
+              "Additional Check 3");
+  test(() => {assert_less_than(baseline4.offsetLeft + baseline4.offsetWidth, 100, "baseline4 and baseline5 should be below baseline6;")}, "Additional Check 4");
```

Comment:
@javifernandez I don't really understand the meaning of this test, tbh. Can you help me figuring out how it works?

Reply:
These checks are evaluating the relative positions of the grid items after being aligned by their respective baselines. For instance, in the last two checks, the expected result is that both, baseline4 and baseline5 elements have then same offsetLeft; this means that both need the same margin to be added to the left side in order to move the elements so that their baselines are aligned with the dominant baseline (in this case named, incorrectly, as baseline6). 

I understand the confusion, but the identifier 'baseline6' in the log message looks like a typo. We could change that by the term "dominant baseline" 

If you think about it, that check is just evaluating that the sum of the margins added to the left are less than the width of the grid container, which means that at least there is a fixed (> 0) amount as right offset (which has to be the same for both)

Reply:
Thank you for the explanation! I will change it to "dominant baseline". What confused me was the fact that the test was comparing baselines in different grids: baseline4 and 5 inside one grid, baseline6 inside another grid.